packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -479,71 +137,6 @@ QUnit.test('properties within an if statement should not fail on re-render', fun
   equal(view.$().text(), '');
 });
 
-QUnit.test('should update the block when object passed to #if helper changes', function() {
```

Comment:
This is a dup of https://github.com/emberjs/ember.js/pull/12914/files#r52059085 as far as I can tell?